project-base/storefront/components/Blocks/Product/Filter/FilterSelectedParameters.tsx

```
@@ -29,173 +31,203 @@ export const FilterSelectedParameters: FC<FilterSelectedParametersProps> = ({ fi
+                                        }}
                                     >
-                                        {selectedValue.rgbHex && selectedValue.rgbHex !== '' && (
-                                            <div
-                                                className="mr-2 h-4 w-4 rounded border border-text"
-                                                style={{ backgroundColor: selectedValue.rgbHex }}
-                                            />
+                                        {currentFilter.minimalPrice !== undefined && (
+                                            <>
+                                                <span>{t('from')}&nbsp;</span>
+                                                {formatPrice(currentFilter.minimalPrice)}
+                                                {currentFilter.maximalPrice !== undefined && <>&nbsp;</>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)